metabrainz/templates/index/about.html

```diff
@@ -3,42 +3,44 @@
-        Commons licenses. Our data is mostly gathered by volunteers, and verified
-        by a voting system to make sure it is consistent and correct. All
-        non-commercial use is free, but commercial users are asked to pay for
-        regular updates of the data in order to help fund the project. We
-        encourage all data users to contribute to the data gathering process so
-        that our data can be as comprehensive as possible.') }}
+        {{ _('The foundation works within a balanced ecosystem of contributions to the datasets 
+        and community and the utilization of the datasets. We encourage all data users to participate 
+        in the data gathering, editing and peer review processes so that our data can be as comprehensive
+        as possible. Each entity is assigned a permanent 
+        <a href="%(MusicBrainz_Identifier_url)s">MBID (MusicBrainz Identifier)</a> 
+        in order to facilitate the creation of a reliable and unambiguous universal lingua franca for music data. 
+        MBID’s have opened the possibility of boundless search queries, exceeding the seven traditional 
+        identifiers for music data and truly putting knowledge at one's fingertips. Our community aggregated 
+        data sets are verified and edited by a global voting system to ensure the data is consistent and correct.
+        We collect information about as many different styles and types of arts and music as possible 
+        and do not discriminate when compiling data.') }}
       </p>
       <p>
-        {{ _('The MetaBrainz Foundation believes in <a href="%(financial_reports_url)s">
-        transparent finances</a> and has several <a href="%(supporters_url)s">
-        commercial supporters</a> who pay for timely and convenient access to
-        data. The MetaBrainz Foundation is also supported by a number of
-        <a href="%(sponsors_url)s">sponsors</a> who provide
-        funds in order for the foundation to accomplish its goals.',
-        financial_reports_url=url_for('financial_reports.index'),
-        supporters_url=url_for('users.supporters_list'),
-        sponsors_url=url_for('index.sponsors')) }}
+        {{ _('The foundation was founded in 2004 by Robert Kaye, in San Luis Obispo, California and is a 
+        501(c)(3) tax-exempt non-profit. <a href="%(supporters_url)s">Commercial users</a> are asked to contribute 
+        financially in order to help fund our family of projects, and give back to our community to keep our 
+        ecosystem healthy. Non-commercial use of our data remains free, and we are committed to keeping it free to utilize. 
+        We are also supported by donations and a number of <a href="%(sponsors_url)s">sponsors</a> who provide 
+        funds in order for the foundation to continue to accomplish various project goals. The MetaBrainz 
+        Foundation believes in<a href="%(financial_reports_url)s">transparent finances,</a> which are published quarterly.
```

Comment:
Make sure to define link values (for placeholders), close the function that was opened with `_(`, and close the template block that was opened with `{{`.